x-pack/plugins/cases/common/constants/index.ts

```diff
@@ -4,34 +4,15 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import { CASE_VIEW_PAGE_TABS } from './types';
-import type { CasesFeaturesAllRequired } from './ui/types';
 
-export const DEFAULT_DATE_FORMAT = 'dateFormat' as const;
-export const DEFAULT_DATE_FORMAT_TZ = 'dateFormat:tz' as const;
-
-/**
- * Application
- */
+import type { CasesFeaturesAllRequired } from '../ui/types';
 
-export const APP_ID = 'cases' as const;
-export const FEATURE_ID = 'generalCases' as const;
-export const APP_OWNER = 'cases' as const;
-export const APP_PATH = '/app/management/insightsAndAlerting/cases' as const;
-export const CASES_CREATE_PATH = '/create' as const;
-export const CASES_CONFIGURE_PATH = '/configure' as const;
-export const CASE_VIEW_PATH = '/:detailName' as const;
-export const CASE_VIEW_COMMENT_PATH = `${CASE_VIEW_PATH}/:commentId` as const;
-export const CASE_VIEW_ALERT_TABLE_PATH =
-  `${CASE_VIEW_PATH}/?tabId=${CASE_VIEW_PAGE_TABS.ALERTS}` as const;
-export const CASE_VIEW_TAB_PATH = `${CASE_VIEW_PATH}/?tabId=:tabId` as const;
+export * from './owners';
```

Comment:
Split this file up to move the file service constants to their own place and moved various other constants to avoid circular references.